languages/elixir/exercises/concept/booleans/.meta/config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "contributors": [
```

Comment:
This can be updated to the [new format](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#metaconfigjson).

Reply:
Already on it!